### DISCUSSION TITLE
Change font of CTkTabview

### DISCUSSION THREAD

**Discussion by Hancie123** [TomSchimansky/CustomTkinter#709] [Q&A] [answered]
How can I change the font size of CTkTabview? Can I frame the frame background color of a specific tab in CTkTabview?

**Comment by TomSchimansky** [comment, marked as answer]
The font can not be configured at the moment, but you can configure a specific frame with:
```tabview.tab("tab_name").configure(...)```
Maybe I will add font attribute to the tabview.

**Comment by PrizeGotti** [comment]
Is there an update on this? It really messes up the look of the UI when things like tab names and frame names don't match the font used on the rest of the UI.

**Comment by mromrell** [comment]
Hi, 2024 here, We would love an update for this. Thanks

**Comment by dipeshSam** [comment]
@Hancie123 , @PrizeGotti  , @mromrell 

You can simply achieve it by updating font of internal `segmented button`. Here are sample function and code:
``` Python
from customtkinter import CTk, CTkTabview
from customtkinter import CTkFont

def apply_font_to_tabs(tabs: CTkTabview, font: CTkFont | tuple):
    """Applies given `font` either tuple or `CTkFont` to tab names."""
    tabs._segmented_button.configure(font=font)

if __name__ == "__main__":
    app = CTk()
    app.geometry("500x300")

    tabs = CTkTabview(app)
    tabs.place(relx=0.5, anchor="center", rely=0.5)

    apply_font_to_tabs(tabs, ("Impact", 15))

    tabs.add("Tab 1")
    tabs.add("Tab 2")
    tabs.add("Tab 3")


    app.mainloop()
```

**Output:**
![ctk_tab_font_change](https://github.com/user-attachments/assets/061a56bb-0400-40cc-a48c-4bf48ad74d01)

Regards
